### DISCUSSION TITLE
Tagging each header in a markdown file

### DISCUSSION THREAD

**Discussion by lamyergeier** [jgm/pandoc#9875] [Ideas]
I have several markdown files. I am struggling with finding relevant content, so I thought that it would be nice if one could tag each header may be (H1 and H2) with several relevant tags so that one could find the relevant section from markdown files easily (from terminal).

It would be useful to store this data in the markdown file itself. Is there possibility that you could support semantic tagging of headers in pandoc?

Any suggestions on what kind of syntax can we use? I was thinking that one could store header tags in the yaml header as array of tags for each header. Is there a better suggestion? Storing data in yaml, is a bit cumbersome as header must be uniquely identified.

**Comment by jgm** [comment]
You can always use attributes, e.g.
```
## My heading {#identifier .class .otherclass key=value}
```
